### DISCUSSION TITLE
Rofi Appimage

### DISCUSSION THREAD

**Discussion by subhaniminhas** [davatorium/rofi#1832] [Ideas/Feature Requests]
Hello dear sir, I love Rofi for it's versatility and theming options, but distro like Debian do not let me use the latest version of rofi out of box. Can we have an Appimage of rofi, which I can then maybe launch from Debian with latest version. 
Regards
Happy Coding

**Comment by DaveDavenport** [comment]
As mentioned in the several issues/feature requests on this:
I have no time to support/maintain any packages (directly or indirectly).
